k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -545,18 +540,11 @@ done
 ## Once Helm Charts are unified, these will become defaulted values.yaml properties.
 default_values_file="../charts/cortx/values.yaml"
 
-cortxdata_service_headless_name=$(yq ".configmap.cortxMotr.headlessServiceName" "${default_values_file}")
-readonly cortxdata_service_headless_name
-
-cortxdata_data_pod_prefix=$(yq ".configmap.cortxMotr.statefulSetName" "${default_values_file}")
-readonly cortxdata_data_pod_prefix
+readonly cortxdata_service_headless_name=cortx-data-headless
 
 cortx_localblockstorage_storageclassname=$(yq ".platform.storage.localBlock.storageClassName" "${default_values_file}")
 readonly cortx_localblockstorage_storageclassname
 
-cluster_domain=$(yq ".configmap.clusterDomain" "${default_values_file}")
-readonly cluster_domain
-
 server_instances_per_node=$(yq ".solution.common.s3.instances_per_node" "${solution_yaml}")
```

Comment:
I missed this when I merged the Server chart. Will address the removal of this in another PR.